docs/standard/class-libraries-multi-targeting.md

```diff
@@ -0,0 +1,339 @@
+---
+title: .NET Class Library Multi-Targeting
+description: Learn how to create .NET class libraries for multiple .NET platforms that can access and wrap platform-specific functionality.
+keywords: .NET, .NET Core
+author: immol
+ms.author: mairaw
+ms.date: 03/19/2018
+ms.topic: article
+ms.prod: .net
+ms.technology: dotnet-standard
+ms.devlang: dotnet
+ms.assetid: 375da522-3520-4735-b1de-68c7b71719d5
+ms.workload: 
+  - "dotnet"
+  - "dotnetcore"
+---
+
+# Introduction
+
+.NET Standard 2.0 offers an extensive API surface that can be shared across .NET
+applications. There are situations where you may need to access platform
+specific APIs that are not part of .NET Standard (for instance, you might want
+to pop UI for starting an OAuth flow). In those cases you often want to build a
+.NET Standard wrapper to encapsulate framework- or OS-specific functionality
+(for example, the geolocation APIs).
+
+You can do this via [multi-targeting](frameworks.md) which builds your project
+for multiple framework from a single project. This enables you to use
+conditional compilation with `#if` to call framework specific APIs.
+
+Please note that this topic is primarily about libraries. For application
+development we generally recommend to use multiple projects (one per
+application) and sharing code via Shared Projects.
+
+# Multi-targeting
+
+.NET Standard projects start as targeting a single framework, namely a specific
+version of .NET Standard:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+</Project>
+```
+
+In order to target multiple frameworks, you need to change the name of the
+property from `TargetFramework` to the plural form `TargetFrameworks`. Then you
+can add all the desired frameworks as a semicolon separated list:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net461;netcoreapp2.0</TargetFrameworks>
```

Comment:
You should say what happens if a proj file has both `TargetFramework` and `TargetFrameworks` elements in it.